_data/tutorials.yaml

```diff
@@ -30,7 +30,8 @@ splitting:
   canonical: confluent
   slug: "/split-a-stream-of-events-into-substreams"
   question: "How do I split events in a Kafka topic so that the events are placed into subtopics?"
-  introduction: "Suppose that you have a Kafka topic representing appearances of an actor or actress in a film, with each event denoting the genre. In this tutorial, we'll write a program that splits the stream into substreams based on the genre. We'll have a topic for drama films, a topic for fantasy films, and a topic for everything else."
+  image: https://raw.githubusercontent.com/confluentinc/event-streaming-patterns/main/docs/img/event-router.svg
```

Comment:
@ybyzek do you know if it's possible to center the image?  I find it a little odd looking at the text above spans a greater width than the image, so i'm wondering if center alignment of the image would look better

Reply:
@rspurgeon ha, it is possible!  But in fact[ I moved it from center to left](https://github.com/confluentinc/kafka-tutorials/pull/925/commits/e6064d4235013c97469e26af6590bc28d93519ff)...but maybe that's not what we want.  Happy to move back